ES/ESbqm005.xml

```diff
@@ -57,40 +57,137 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                     </msItem>
+                     <msItem xml:id="ms_i1.3">
+                        <locus from="8ra" to="13vb"/>
+                        <title type="complete" ref="LIT1544Gebrah#PalmSundayGH">Palm Sunday</title>
+                     </msItem>                     
+                     <msItem xml:id="ms_i1.4">
+                        <locus from="13vb" to="27va"/>
+                        <title type="complete" ref="LIT1544Gebrah#Monday">Monday</title>                        
+                        <msItem xml:id="ms_i1.4.1">
+                           <locus from="19rb" to="20ra"/>
+                           <title ref="LIT1544Gebrah#JohnMonday">Homily by John Chrysostom for Monday morning</title>                           
+                        </msItem>
+                        <msItem xml:id="ms_i1.4.2">
+                           <locus from="20ra" to="22va"/>
+                           <title ref="LIT2122OntheF">Homily by John Chrysostom on the fig tree for Monday morning</title>                           
+                        </msItem>
+                     </msItem>                 
+                     <msItem xml:id="ms_i1.5">
+                        <locus from="27va" to="40va"/>
+                        <title type="complete" ref="LIT1544Gebrah#Tuesday">Tuesday</title>
+                     </msItem>                     
+                     <msItem xml:id="ms_i1.6">
+                        <locus from="40va" to="50rc"/>
+                        <title type="complete" ref="LIT1544Gebrah#Wednesday">Wednesday</title>
+                     </msItem>  
+                     <msItem xml:id="ms_i1.7">
+                        <locus from="50rc" to="66va"/>
+                        <title type="complete" ref="LIT1544Gebrah#Thursday">Thursday</title>         
+                        <note>- On <locus target="#50rc">50rc</locus>, the 1st night hour of Thursday
+                           (Text I-7) is wrongly indicated as the 11th night hour.</note>
+                     </msItem>                     
+                     <msItem xml:id="ms_i1.8">
+                        <locus from="67ra" to="127va"/>
+                        <title type="complete" ref="LIT1544Gebrah#Friday">Friday</title>
+                        <msItem xml:id="ms_i1.8.1">
+                           <locus from="72rc" to="79rb"/>
+                           <title type="complete" ref="LIT2142OntheS"/>                           
+                        </msItem>
+                        <msItem xml:id="ms_i1.8.2">
+                           <locus from="92ra" to="97va"/>
+                           <title type="complete" ref="LIT1180Baenta"/>
+                        </msItem>
+                        <msItem xml:id="ms_i1.8.3">
+                           <locus from="97va" to="100va"/>
+                           <title type="complete" ref="LIT5810HomilySatan">Homily by Ephrem on Satan and on the death</title>
```

Comment:
the same old typos in the title here and later - it might be faster to delete the titles' content, since it is present in the work records anyway.